[bitnami/wavefront-proxy] Add deprecation notice (#37754)
bitnami/containers: bitnami/wavefront-proxy/README.md

# Deprecation notice

> The Wavefront developers have deprecated the chart installation method for Wavefront, instead now using the Kubernetes Operator. So this container is going to be deprecated.

Reviewer: ```suggestion
Remove Wavefront chart since this installation method has been deprecated in favor of the Kubernetes Operator.
```
